frontend/apps/web/src/pages/all-tests/index.tsx

```diff
@@ -0,0 +1,285 @@
+import { PageContainer } from '@/containers/PageContainer'
+import {
+  Button,
+  Colors,
+  Link,
+  PageLayout,
+  ProgressBar,
+  Table,
+  Text,
+  usePageLayout,
+} from '@filecoin/ui'
+import { styled } from '@storybook/theming'
+import React from 'react'
+import { useNavigate } from 'react-router-dom'
+
+const Dashboard = () => {
+  const {
+    state: { model },
+  } = PageContainer.useContainer()
+  const allTests = model.getAllTests()
+  const navigate = useNavigate()
+
+  const Header = props => {
```

Comment:
Since the is no dependency in Header, you can move it outside of Dashboard function.

Reply:
Done